roles/vdm/templates/resources/platform-postgres-dataserver.yaml

```diff
@@ -0,0 +1,35 @@
+{%- set db = db_default_name_map[role] if ('database' not in settings and role in db_default_name_map) else settings.database|default(role, true) -%}
+{%- set server_name = role if role != "default" else "postgres" -%}
```

Comment:
Why this condition based on role? Isn't this file only used in case of `role != "default"`? I am guessing role will never be default, am I missing something?
https://github.com/sassoftware/viya4-deployment/blob/7f7d457459d522f20b5c21fd38ef3e62ec788196/roles/vdm/tasks/postgres/postgres-instance.yaml#L147

Reply:
I was reusing the working logic from https://github.com/sassoftware/viya4-deployment/blob/main/roles/vdm/templates/resources/postgres-instance.yaml which was from Crunchy 4, I could probably update that to `{%- set server_name = role -%}`

Reply:
Marking as resolved, this change was put in.